source/stylesheets/base/_typography.scss

```diff
@@ -39,26 +39,43 @@ a {
   }
 }
 
-.link-light {
-  color: $blue;
-  background: url('../../images/link-background-light.svg') no-repeat center left;
-  background-size: contain;
-  padding: 3px 20px;
-}
-
-.link-medium {
-  color: $blue;
-  background: url('../../images/link-background-medium.svg') no-repeat center center;
+@mixin gen-link($color, $imgURL, $position, $imgURLHover) {
```

Comment:
I'd name your argument variables with lowercase and hyphens. For instance, `$img-url` and `img-url-hover`. It's a nitpick but this syntax feels out of place in a Sass file. It feels more javascript-y.

Reply:
I'd be more long-winded when it comes to the mixin name. I'm guessing this means generate-link. But you can just say that. It's more a more clear name for someone who might come after you. It will be easier for them to decipher your code.